elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/client/impl/BackendRequest.java

```diff
@@ -69,6 +71,13 @@ public String getIndexName() {
 		return indexName;
 	}
 
+	/**
+	 * Whether performing an explicit index refresh after executing this action is needed or not.
+	 */
+	public boolean needsRefreshAfterWrite() {
```

Comment:
This method is never used.
Wondering if you had plans, or forgot to commit some tests?